front-end/lambda-in/src/Components/Landing Page/Navbar/navbar.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+//import { Navbar, NavItem, NavDropdown, Nav, MenuItem } from 'react-bootstrap';
+import './navbar.css';
+import Button from '@material-ui/core/Button';
+import Paper from '@material-ui/core/Paper';
+
+class NavBar extends React.Component {
```

Comment:
I would suggest naming the file in CamelCase, so the file will match the Class name.
```
NavBar.js
```